arangod/RocksDBEngine/RocksDBKey.h

```diff
@@ -31,6 +31,7 @@
 #include "VocBase/voc-types.h"
 #include "Zkd/ZkdHelper.h"
 
+#include <iosfwd>
```

Comment:
Needed?

Reply:
I think so, as the file is implementing `operator<<` to append to an `std::ostream`.
It already did so before, but the required headers were missing.